### PR TITLE
:wink: add ninja build excutable

### PR DESCRIPTION
**Pull request by Lambourl**
the purpose of this Pr is to add ninja to our tools

### CODE REVIEW

**Review by daminetreg — APPROVED** [review]
Simple, effective.

File: distro.json [hunk 1]
```diff
@@ -69,9 +69,9 @@
       "platforms": {
         "all": {
           "universal": {
-            "url": "https://github.com/tipi-build/environments/archive/94ecd285bbc7fd74504e721799d43716d2560828.zip",
-            "sha1": "bcd83a12d4232f9ed75c732002cdee8606965157",
-            "root": "environments-94ecd285bbc7fd74504e721799d43716d2560828"
+            "url": "https://github.com/tipi-build/environments/archive/99b29c25efce9cc6445a857262e6536659aa5777.zip",
+            "sha1": "8823f84eb432b4b619e8af41cfeca9c143d7d164",
+            "root": "environments-99b29c25efce9cc6445a857262e6536659aa5777"
```
Comment (daminetreg) [inline distro.json:74]: This requires changes there are obvious issue in the environments fiels.

Comment (daminetreg) [reply distro.json:74]: See my review on https://github.com/tipi-build/environments/pull/21

**Review by daminetreg — CHANGES_REQUESTED** [review]

**Review by pysco68 — APPROVED** [review]
LGTM

**Pull request merged**